indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -48,7 +48,7 @@
             {% endif %}
         </div>
         <div class="toolbar">
-            {% if registration.registration_form.modification_mode.name != 'not_allowed' and registration.is_active %}
+            {% if registration.can_be_modified and registration.registration_form.modification_mode.name != 'not_allowed' and registration.is_active %}
```

Comment:
I don't think adding this check here is correct. We have cases that disable and show detailed tooltips within this if block exactly in case the registration cannot be modified.